src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -547,8 +548,18 @@ export function transform(formConfig, form) {
     delete clonedData.additionalIncidentText;
     delete clonedData.additionalSecondaryIncidentText;
   }
+  let attachments = [];
 
-  return JSON.stringify({ form526: clonedData });
+  // Flatten all attachment pages into attachments ARRAY
+  ATTACHMENT_KEYS.forEach(key => {
```

Comment:
This was the "cleanest" way I could think of making this work. It could maybe be improved upon